include/cantera/oneD/StFlow.h

```diff
@@ -292,8 +292,21 @@ class StFlow : public Domain1D
         return m_rho[j];
     }
 
+    //! @deprecated To be removed after %Cantera 3.0. Superseded by isFree()
     virtual bool fixed_mdot();
 
+    //! Retrieve flag indicating whether flow is free
+    //! @since New in %Cantera 3.0
+    bool isFree() const {
+        return m_isFree;
+    }
+
+    //! Retrieve flag indicating whether flow uses Lambda
+    //! @since New in %Cantera 3.0
+    bool usesLambda() const {
```

Comment:
I think this would be a good place to add a bit of documentation about what it means for the flow to "use lambda". That is, `usesLambda == true` means that we solve the radial momentum equation and the equation $d\Lambda/dz = 0$, and `usesLambda == false` means we solve the trivial equations $\Lambda(z) = 0$ and $V(z) = 0$.

Reply:
Added documentation. Also renamed to `isStrained` to have better symmetry with `isFree`.